functions/icons.zsh

```diff
@@ -213,22 +213,22 @@ case $POWERLEVEL9K_MODE in
       VCS_COMMIT_ICON                $'\uE729 '             # 
       VCS_BRANCH_ICON                $'\uF126 '             # 
       VCS_REMOTE_BRANCH_ICON         $'\uE728 '             # 
-      VCS_GIT_ICON                   $'\uF113 '             # 
+      VCS_GIT_ICON                   $''                    # 
       VCS_GIT_GITHUB_ICON            $'\uE709 '             # 
       VCS_GIT_BITBUCKET_ICON         $'\uE703 '             # 
       VCS_GIT_GITLAB_ICON            $'\uF296 '             # 
-      VCS_HG_ICON                    $'\uF0C3 '             # 
-      VCS_SVN_ICON                   $'\uE72D '             # 
+      VCS_HG_ICON                    $''                    # 
```

Comment:
Why was the HG Icon removed? Okay, it is not the official Logo, but its better than nothing.
![bildschirmfoto 2018-04-21 um 15 36 24](https://user-images.githubusercontent.com/1544760/39084770-58796b82-457a-11e8-864e-6066b4390beb.png)